uy.com.catuy.framework.core.exception.test.DEFAULT_MESSAGE=No fue posible obtener el detalle de la excepcion.
